routes/chat.js

```diff
@@ -122,6 +123,8 @@ router.get(
   registerShareGif
 );
 
+router.delete('/message/:messageID', auth.isAuth, deleteMessage);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The route for deleting a message is well defined and uses the correct HTTP verb `DELETE`. However, it would be beneficial to add some validation middleware to check if `messageID` is provided in the URL parameters. This will ensure that the server doesn't attempt to delete a message without an ID, which could lead to unexpected behavior or errors.
```diff
- router.delete('/message/:messageID', auth.isAuth, deleteMessage);
+ router.delete('/message/:messageID', validate(ChatValidation.deleteMessage), auth.isAuth, deleteMessage);
```
In the above code, `ChatValidation.deleteMessage` should be a middleware function that validates the presence and format of `messageID`. If the validation fails, it should send an appropriate error response.


<!-- This is an auto-generated comment by OSS CodeRabbit -->